client/src/types.ts

```diff
@@ -120,6 +120,11 @@ export interface Answer {
   media?: Media[];
 }
 
+export enum AnswerAttentionNeeded {
+  NONE = "NONE",
```

Comment:
let's never never ever use a string literal value like `NONE` if it's an option to use the empty string